app/models/common_exposure_cohort.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# CommonExposureCohort: a common exposure cohort
+class CommonExposureCohort < ApplicationRecord
+  belongs_to :patient, touch: true
+
+  VALID_COHORT_TYPES = [nil, '', 'Adult Congregate Living Facility', 'Child Care Facility', 'Community Event or Mass Gathering', 'Correctional Facility',
```

Comment:
For consistency with the patient, do we also want to validate that these strings don't get longer than 200 characters.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/11f9765d3d0d6b687f0a3e8f6d62ab6192eadf4b#diff-f28872be876de2e68f61749a7ff939895bf99f0e8546b70b1770c495fc0b0508R11